src/GitLink/Linker.cs

```diff
@@ -131,5 +162,41 @@ public static bool Link(string pdbPath, LinkOptions options = default(LinkOption
 
             return true;
         }
+
+        private static void CreateSrcSrv(string srcsrvFile, SrcSrvContext srcSrvContext)
+        {
+            Argument.IsNotNull(() => srcSrvContext);
+            Argument.IsNotNullOrWhitespace(() => srcSrvContext.RawUrl);
+            Argument.IsNotNullOrWhitespace(() => srcSrvContext.Revision);
+            Argument.IsNotNullOrWhitespace(() => srcsrvFile);
+
+            if (srcSrvContext.VstsData.Count != 0)
+            {
+                File.WriteAllBytes(srcsrvFile, SrcSrv.CreateVsts(srcSrvContext.Revision, srcSrvContext.Paths, srcSrvContext.VstsData));
+            }
+            else
+            {
+                File.WriteAllBytes(srcsrvFile, SrcSrv.Create(srcSrvContext.RawUrl, srcSrvContext.Revision, srcSrvContext.Paths, srcSrvContext.DownloadWithPowershell));
+            }
+        }
+
+        private static string GetNormalizedPath(string path, string gitRepoRootDir)
```

Comment:
An extension method on what? string? I think a private static utility method that serves only one class is best stored in the class that consumes it. Making it an extension method would add it to every single string object, most instances of which don't represent paths. 
Moving it to an IO/path related utility class and making it internal static instead, sure, we could do that. Is that what you meant?